turbopack/crates/turbopack-ecmascript/tests/tree-shaker/analyzer/next-response/output.md

```diff
@@ -401,182 +401,47 @@ graph TD
 # Final
```

Comment:
Original:
<img width="1046" alt="image" src="https://github.com/user-attachments/assets/07058cc7-feb7-4cce-972d-d77a00418b90" />



New: 
<img width="1046" alt="image" src="https://github.com/user-attachments/assets/84a1acba-945b-47ee-a5c5-16ee5c49b5d7" />


Now the module evaluation is gone, many nodes could be merged